infra/modules/dms/main.tf

```
@@ -0,0 +1,90 @@
+# Put IAM Roles here
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "aws_iam_policy" "dms_access" {
+  name   = "dms-access"
+  policy = data.aws_iam_policy_document.dms_access.json
+}
+resource "aws_iam_role" "dms_access" {
+  name               = "dms-access-role"
+  assume_role_policy = data.aws_iam_policy_document.dms-assume-role-policy.json
+}
+
+data "aws_iam_policy_document" "dms-assume-role-policy" {
```

Suggest underscores for consistency:

```suggestion
  assume_role_policy = data.aws_iam_policy_document.dms_assume_role_policy.json
}

data "aws_iam_policy_document" "dms_assume_role_policy" {
```